py3status/modules/static_string.py

```diff
@@ -3,35 +3,31 @@
 Display static text.
 
 Configuration parameters:
-    format: text that should be printed (default '')
+    format: default format for this module (default 'hello, world!')
```

Comment:
the thing is it is the format not the default so the description should reflect that so
```
format: display format for this module (default 'Hello, world!')
```